package.json

```diff
@@ -58,6 +58,7 @@
     "styleguide/dist"
   ],
   "jest": {
+    "testEnvironment": "node",
```

Comment:
@nnnnat I added this line to package.json. Without it, every test was failing with "SecurityError: localStorage is not available for opaque origins".

Reply:
I'm wondering if that error was being thrown cause you're using the `window` object inside the component. If you wrap the `window` in a conditional that checks for the `window` object first this might not be necessary.